lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -677,15 +672,44 @@ struct ziti_instance_s *new_ziti_instance(const char *identifier) {
 }
 
 int init_ziti_instance(struct ziti_instance_s *inst, const ziti_config *cfg, const ziti_options *opts) {
+    FREE(inst->ztx);
```

Comment:
this seems like a strange, "first thing" to do, but i assume it's on purpose? just want to make sure it is?

Reply:
yeah, unless the caller is aware that a ziti_context is allocated by init_ziti_instance and goes out of their way to free it, calling more than once with the same ziti_instance will leak ztx.

Reply:
i see. why not just return an error? if i call it twice and stored off that context thinking i was making a second one that might be 'surprising'?